Packs/SOCRadar/Integrations/FeedSOCRadarThreatFeed/FeedSOCRadarThreatFeed.py

```diff
@@ -0,0 +1,432 @@
+        'hash': FeedIndicatorType.File,
+    }[socradar_indicator_type]
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with SOCRadar Threat Intelligence API. Overrides BaseClient.
+    """
+
+    def __init__(self, base_url, api_key, tags, tlp_color, verify, proxy):
+        super().__init__(base_url, verify=verify, proxy=proxy)
+        self.api_key = api_key
+        self.tags = tags
+        self.tlp_color = tlp_color
+
+    def get_collection_indicators(self, collection_name, offset, limit):
+        suffix = '/threat/intelligence/socradar_collections'
+        api_params = {'key': self.api_key, 'collection_names': [collection_name], 'limit': limit, 'offset': offset}
+        response = self._http_request(method='GET', url_suffix=suffix, params=api_params, timeout=60,
+                                      error_handler=self.handle_error_response)
+        return response
+
+    def check_auth(self):
+        suffix = '/threat/intelligence/check/auth'
+        api_params = {'key': self.api_key}
+        response = self._http_request(method='GET', url_suffix=suffix, params=api_params,
+                                      error_handler=self.handle_error_response)
+
+        return response
+
+    def parse_raw_indicators(self, raw_indicators: list, collection_feed_type: str) -> list:
+        """Creates a list of indicators from a given response
+
+        :type raw_indicators: ``list``
+        :param raw_indicators: List of dict that represent the response from the api
+
+        :type collection_feed_type: ``str``
+        :param collection_feed_type: Type of the indicators that exist in the collection
+
+        :return: List of indicators with the correct indicator type.
+        :rtype: ``list``
+        """
+        parsed_indicators = []
+
+        collection_indicator_type = convert_to_demisto_indicator_type(collection_feed_type)
+
+        for indicator_dict in raw_indicators:
+            if indicator_dict:
+                indicator = indicator_dict.get('feed', '')
+                indicator_type = convert_to_demisto_indicator_type(
+                    indicator_dict.get('feed_type', ''), indicator) or collection_indicator_type
+                if not indicator_type:
+                    indicator_type = auto_detect_indicator_type(indicator)
+
+                first_seen_date = indicator_dict.get('first_seen_date', '')
+                last_seen_date = indicator_dict.get('latest_seen_date', '')
+                maintainer_name = indicator_dict.get('maintainer_name', '')
+                extra_info = indicator_dict.get('extra_info', {})
+
+                indicator_obj = {
+                    "type": indicator_type,
+                    "value": indicator,
+                    "rawJSON": {
+                        "value": indicator,
+                        "type": indicator_type
+                    },
+                    "fields": {
```

Comment:
Deleted mapping since it wasn't essential for the integration. Initially, it was created just to be a default selection for the mapper object in the adding new integration window. So there is no need to add it to the fields here since it won't make any difference.